README.md

```diff
@@ -39,20 +39,34 @@ It will serve the following Composer repository at `/packages.json` automagicall
 
 ## Installation
 
-Create the project:
+### 1. Create the project:
 
 ```
 composer create-project rarst/release-belt
 ```
 
-Place release ZIPs into `/releases/[type]/[vendor]/`.
+### 2. Place release ZIPs into `/releases/[type]/[vendor]/`. 
+`[type]` could be e.g. "symfony-bundle", "wordpress-plugin or "typo3-cms-extension".
 
-Configure a web server to serve `index.php`, for example with the following `.htaccess`:
+### 3. Configure a web server to serve `index.php`
+
+For example with the following `.htaccess`:
 
 ```
 FallbackResource /index.php
 ```
 
+If your server does not support FallbackResource, you can use mod_rewrite in your `.htaccess` with this code:
+
+```
+RewriteEngine On
```

Comment:
Please wrap these directives in a check if `mod_rewrite` is available, or they would crash the site for someone trying them without it compiled into Apache.